src/common/i18n.tsx

```diff
@@ -105,7 +107,8 @@ export function I18nProvider(props: RenderableProps<I18nProviderProps>) {
     }, [isLoading, props.locale]);
   }
 
-  const value = useMemo(() => ({ t, setLocale }), [t, setLocale]);
+  const langTag = useMemo(() => t('lang_tag'), [t, props.locale])
+  const value = { t, setLocale, langTag };
```

Comment:
I wonder if we should still memo-ize value? If the identity of the value prop on i18nContext.Provider changes on each render, I wonder if that will lead to unnecessary re-renders?

Reply:
Yeah, I was kind of wondering about this, but wasn't sure. You're probably right on the re-renders.

Reply:
Yeah, I'm not sure either. It's all premature optimization anyway since we don't actually trigger re-renders yet 😅